### PR TITLE
Add NMAE and NMBE, deadband discussion to metrics page

### PR DESCRIPTION
**Pull request by dplarson**
Resolves issue #141 by adding definitions of the normalized MAE (NMAE) and normalized MBE (NMBE) to the metrics page, including links in the sidebar menu. Also adds notes to the NMAE, NMBE, and NRMSE definitions about how the norm value is defined.

### CODE REVIEW

**Comment by dplarson** [comment]
@wholmgren Ready for review

**Comment by wholmgren** [comment]
Thanks @dplarson. This looks good. Let's leave it open pending resolution of what to do with other variables as discussed in https://github.com/SolarArbiter/solarforecastarbiter-core/issues/370

**Comment by cwhanse** [comment]
LGTM also

File: metrics.md [hunk 1]
```diff
@@ -64,11 +64,30 @@ The absolute percentage error is the absolute value of the difference between th
...
+The NMAE [%] is the normalized form of the MAE and is defined as:
+
+$$ \text{NMAE} = \frac{100\%}{\text{norm}} \cdot \frac{1}{n} \sum_{i=1}^n  \lvert F_i - O_i \rvert $$
+
+where norm is a constant upper bound on the value of the forecasted variable, e.g., 1000 W/m$$^2$$ for GHI or the nameplate AC capacity of a PV plant.
+
+
+### Normalized Mean Bias Error (NMBE) {#nmbe}
+{: .anchor }
+The NMBE [%} is the normalized form of the MBE and is defined as:
```
Comment (alorenzo175) [inline metrics.md]:
```suggestion
The NMBE [%] is the normalized form of the MBE and is defined as:
```

**Comment by wholmgren** [comment]
@dplarson can you update this PR in light of discussions in 

* https://github.com/SolarArbiter/solarforecastarbiter-core/pull/379
* https://github.com/SolarArbiter/solarforecastarbiter-core/issues/381
* https://github.com/SolarArbiter/solarforecastarbiter-core/pull/378

Key points, I think:

* Currently no user control over normalization via dashboard. 
* AC (DC) power normalized by AC (DC) capacity.
* Irradiance and other weather variables currently have `nan` normalization factors so metrics are also `nan`
* Need some kind of uncertainty/deadband discussion on metrics page. Take from text already written in docs, report table as much as possible.

**Comment by dplarson** [comment]
Yes, I'll push updates today for those items.

**Comment by dplarson** [comment]
@wholmgren Thoughts on the new wording regarding 1) normalized metrics and 2) deadbands?

**Pull request merged**